test/component/uk/gov/hmrc/individualsdetailsapi/controllers/SandboxAddressesControllerSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package component.uk.gov.hmrc.individualsdetailsapi.controllers
+
+import component.uk.gov.hmrc.individualsdetailsapi.stubs.{AuthStub, BaseSpec}
+import play.api.test.Helpers._
+import scalaj.http.Http
+
+class SandboxAddressesControllerSpec extends BaseSpec {
+  val rootScope = "read:individuals-details-contact-details"
```

Comment:
Should this not match the live controller for consistency? 

List("read:individuals-details-hmcts-c4", "read:individuals-details-laa-c4")

Reply:
Also do we need to update the sandbox contact details details controller?